<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvancedSettings</class>
 <widget class="QWidget" name="AdvancedSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1084</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="5,0" columnstretch="1,0">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="maximumSize">
      <size>
       <width>325</width>
       <height>250</height>
      </size>
     </property>
     <property name="title">
      <string>Application Settings</string>
     </property>
     <widget class="QPushButton" name="pushButton_CheckUpdates">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>160</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>25</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>Check for Software Updates</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_CheckFirmware">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>64</y>
        <width>160</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>Check for Firmware Update</string>
      </property>
     </widget>
     <widget class="QLabel" name="text_USBExchangePeriod">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>208</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>USB exchange period</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignHCenter</set>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_USBExchangePeriod">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>208</y>
        <width>50</width>
        <height>19</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
      <property name="correctionMode">
       <enum>QAbstractSpinBox::CorrectionMode::CorrectToNearestValue</enum>
      </property>
      <property name="suffix">
       <string> ms</string>
      </property>
      <property name="minimum">
       <number>2</number>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>5</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_RestartApp">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>97</y>
        <width>161</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>25</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Restart for full changes</string>
      </property>
      <property name="text">
       <string>Restart Application</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_About">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>161</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string>About HOTAS Configurator</string>
      </property>
     </widget>
     <widget class="QLabel" name="text_FontSize">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>81</width>
        <height>19</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Font size</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_FontSize">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>178</y>
        <width>50</width>
        <height>19</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
      <property name="minimum">
       <number>7</number>
      </property>
      <property name="maximum">
       <number>12</number>
      </property>
      <property name="value">
       <number>8</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_LangDeutsch">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>130</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string notr="true">Deutsch</string>
      </property>
      <property name="icon">
       <iconset resource="../../resources.qrc">
        <normaloff>:/Images/flags/Germany.png</normaloff>:/Images/flags/Germany.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_LangEnglish">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>29</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string notr="true">English</string>
      </property>
      <property name="icon">
       <iconset resource="../../resources.qrc">
        <normaloff>:/Images/flags/United_Kingdom.png</normaloff>:/Images/flags/United_Kingdom.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_LangRussian">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>64</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string notr="true">Русский</string>
      </property>
      <property name="icon">
       <iconset resource="../../resources.qrc">
        <normaloff>:/Images/flags/Russia.png</normaloff>:/Images/flags/Russia.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_LangSChinese">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>97</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>25</height>
       </size>
      </property>
      <property name="text">
       <string notr="true">简体中文</string>
      </property>
      <property name="icon">
       <iconset resource="../../resources.qrc">
        <normaloff>:/Images/flags/china.png</normaloff>:/Images/flags/china.png</iconset>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Firmware flasher</string>
     </property>
     <layout class="QHBoxLayout" name="layoutH_Flasher">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Device Information</string>
     </property>
     <widget class="QLineEdit" name="line_DeviceModel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_DeviceSerial">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_DeviceDoM">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="line_DeviceFwVersion">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>23</y>
        <width>37</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>Model:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>63</y>
        <width>37</width>
        <height>12</height>
       </rect>
      </property>
      <property name="text">
       <string>Serial #:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>101</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Date of Manufacture:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>141</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Firmware Version:</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
